src/test/java/com/clevertap/maven/plugins/supertest/SuperTestMavenPluginTest.java

```diff
@@ -0,0 +1,64 @@
+package com.clevertap.maven.plugins.supertest;
+
+import org.junit.jupiter.api.Test;
+import org.xml.sax.SAXException;
+
+import javax.xml.parsers.ParserConfigurationException;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static junit.framework.Assert.assertEquals;
+
+class SuperTestMavenPluginTest {
+    @Test
+    void createRerunCommandTest() throws IOException, ParserConfigurationException, SAXException  {
+        SuperTestMavenPlugin bv = new SuperTestMavenPlugin();
+        final String classXML1 = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n"
```

Comment:
No long strings. Please create files in the resources directory, and load them here instead.

Reply:
got it. Will make the change.